dotcom-rendering/playwright/tests/parallel-1/article.e2e.spec.ts

```diff
@@ -0,0 +1,121 @@
+import type { Response } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { disableCMP } from '../../lib/cmp';
+import { waitForIsland } from '../../lib/islands';
+import { loadPage } from '../../lib/load-page';
+
+const responseHasJsonProperty = async (
+	response: Response,
+	url: string | RegExp,
+	expectedProperty: string,
+): Promise<boolean> => {
+	const isURL = response.request().url().match(url);
+	if (!isURL) return false;
+	const status = response.status();
+	if (status !== 200) return false;
+	const json = (await response.json()) as Record<string, string>;
+	const prop = json[expectedProperty];
+	if (!prop) return false;
+	return true;
+};
+
+test.describe('E2E Page rendering', () => {
+	test.describe('for WEB', () => {
+		test('It should load an article and make the expected network requests for dynamic content', async ({
+			context,
+			page,
```

Comment:
Each test in Playwright provides `page` and `context` objects which is how we interact with the browser context and page in the test itself.